acceptance_tests/utilities/collex_and_survey_helper.py

```diff
@@ -2,33 +2,31 @@
 from datetime import datetime
 
 import requests
+
 from config import Config
 
 
-def add_survey(context, sample_validation_rules):
-    context.survey_id = str(uuid.uuid4())
-    context.survey_name = 'test survey ' + datetime.now().strftime("%m/%d/%Y, %H:%M:%S")
+def add_survey(sample_validation_rules):
```

Comment:
Minor comment - is it worth splitting up this file into something like `collex_helper` and `survey_helper` now?